lib/api/github.js

```diff
@@ -59,8 +59,10 @@ async function initRepo(repoName, token, endpoint) {
 async function findFilePaths(fileName) {
   const res = await ghGot(`search/code?q=repo:${config.repoName}+filename:${fileName}`);
   const exactMatches = res.body.items.filter(item => item.name === fileName);
-  const filePaths = exactMatches.map(item => item.path);
-  return filePaths;
+
+  // GitHub seems to return files in the root with a leading `/`
+  // which then breaks things later on down the line
+  return exactMatches.map(item => item.path.replace(/^\//, ''));
 }
```

Comment:
Actually I will merge this now before hearing your answer, as part of this PR is what I'd call a critical fix and I don't want to delay it.